packages/smart-wallet/src/smartWallet.js

```diff
@@ -164,6 +211,16 @@ export const prepareSmartWallet = (baggage, shared) => {
 
   const makeRecorderKit = prepareRecorderKit(baggage, shared.publicMarshaller);
 
+  const BRAND_TO_PURSES_KEY = 'brandToPurses';
```

Comment:
consider module scope.

eventually we should have some baggage schema data structure this could reference.

Reply:
oops! yeah; that's supposed to be exported, in fact.

I even considered adding some random characters to mitigate collision risk.